swagger/src/main/scala/org/http4s/rho/swagger/TypeBuilder.scala

```diff
@@ -309,4 +307,13 @@ object TypeBuilder {
     private[this] def isCollection(t: Type): Boolean =
       t <:< typeOf[collection.Traversable[_]] || t <:< typeOf[java.util.Collection[_]]
   }
+
+  implicit class WrappedType(val t: Type){
```

Comment:
Mark this private so it won't leave this file... I usually try to avoid implicit conversions like this, but I understand the utility here so its fine as long as its localized.